proposals/lip-0049.md

```diff
@@ -456,7 +456,15 @@ The role of the channel terminated message is to inform chains that their chain
 
 ##### Parameters
 
-The `params` schema for the channel terminated CCM is the empty schema.
+The `params` schema for the channel terminated CCM is the empty schema:
```

Comment:
I find the formulation slightly wrong because the schema is not empty as it uses 3 keywords. I would say 

> The `params` for the channel terminated CCM do not have any properties, i.e. the following schema is used:

Or

> The `params` schema for the channel terminated CCM is the following one without any properties:

Reply:
I aligned with other ccm params description